.github/workflows/formatting-linting.yml

```diff
@@ -3,6 +3,7 @@ name: 'formatting & linting'
 on:
   workflow_dispatch:
   workflow_call:
+  push:
```

Comment:
Nice catch. Added https://aragonassociation.atlassian.net/browse/OS-846 to our task list to further improve it